client/allocrunner/taskrunner/getter/util_linux.go

```diff
@@ -75,52 +75,58 @@ func lockdown(allocDir, taskDir string) error {
 
 func additionalFilesForVCS() []*landlock.Path {
 	const (
-		sshDir        = ".ssh"                  // git ssh
-		knownHosts    = ".ssh/known_hosts"      // git ssh
-		etcPasswd     = "/etc/passwd"           // git ssh
-		gitGlobalFile = "/etc/gitconfig"        // https://git-scm.com/docs/git-config#SCOPES
-		hgGlobalFile  = "/etc/mercurial/hgrc"   // https://www.mercurial-scm.org/doc/hgrc.5.html#files
-		hgGlobalDir   = "/etc/mercurial/hgrc.d" // https://www.mercurial-scm.org/doc/hgrc.5.html#files
+		homeSSHDir     = ".ssh"                     // git ssh
+		homeKnownHosts = ".ssh/known_hosts"         // git ssh
+		etcPasswd      = "/etc/passwd"              // git ssh
+		etcKnownHosts  = "/etc/ssh/ssh_known_hosts" // git ssh
```

Comment:
Should all of `/etc/ssh` be in here maybe? I'm looking at my own machine and I've got a `/etc/ssh/ssh_config` file and a `/etc/ssh/ssh_config.d/` directory, and that's distinct from `/etc/sshd`